conf/rest/9.12.0/qtree.yaml

```diff
@@ -20,7 +22,8 @@ endpoints:
       - ^^qtree           => qtree
       - ^status           => status
       - ^oplock_mode      => oplocks
-
+      - filter:
+          - qtree=!""
```

Comment:
It won't be required as endpoint code flow will add labels if matched but it won't do instance creation